packages/tools/guiEditor/src/diagram/workbench.tsx

```diff
@@ -1041,7 +1041,16 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
         } else if (event.detail) {
             delta = -event.detail;
         }
+        const mouseToRtt = CoordinateHelper.MousePointerToRTTSpace(this.trueRootContainer, this._scene.pointerX, this._scene.pointerY);

```

Comment:
It seems to me like the zoom in is correct but the zoom out no longer respects the mouse position, but instead the previous zoom position on mouse in. I think the improved accuracy of the zoom in experience is coloring the zoom out experience. To repro, mouse over one of the buttons in the snapshot scene and zoom in. The button stays in view the whole time. Then move the mouse to a different location on screen before zooming back out. The zoom seems detached from the cursor and seems to differ from the zoom in experience.

Reply:
@PatrickRyanMS so it should also move to the cursor position when zooming out? I didn't add that because I thought it could be confusing on the zoom out case 🤔

Reply:
we are training users that the cursor is the center the zoom when increasing zoom. If we then change that the center of zoom is at the previous zoom point, what happens when the user zooms in, then pans, then zooms out again? We should just be consistent and make the cursor the center of the zoom in all cases. Most of the time the user's eyes will be near the cursor when actively using the mouse, so it would make sense for the zoom to be centered on the user's sightline.

Reply:
Hey @PatrickRyanMS ! I updated the behavior, can you check it out?

Reply:
@PatrickRyanMS  - Please resolve this conversation if the behavior is correct. This way it will be merged automatically.

Reply:
Zoom in/out is much better now.